machine-learning/solutions/seismic_salt_detection/trainer/model.py

```diff
@@ -73,51 +73,40 @@ def create_regressor(config, parameters):
     Returns:
         A configured and ready to use `tf.estimator.DNNRegressor`
```

Comment:
What makes an estimator "configured"?  The docstrings in this module haven't, in general, been very helpful to me.  They feel a little like they were written so that the file would have docstrings.

If it's because I don't have the TensorFlow experience that most people will have (and this file is assuming), that's ok, too.  In that case, I think I recommend just using the single-line short form.

Reply:
configured = "estimator instance with specified columns, architecture, metrics, optimizer, etc"

Initially my docstrings were much bigger/descriptive but Brian complained a lot about them as he said I was just describing the logic of the code and doc strings shouldn't be that.

I'm open to change this to anything you suggest.